manifests/config.pp

```diff
@@ -244,13 +249,6 @@
     notify => Class['keycloak::service'],
   }
 
-  file { $_server_conf_dir:
```

Comment:
I'd recommend keeping this resource and only using `$_dirs` for domain since that's where the path is deeply nested. Also the permissions on this resource are more appropriate for configs.